ui/templates/includes/taxonomy_panel.html

```diff
@@ -1,24 +1,11 @@
 <div class="cd-panel-2 from-right">
   <header class="cd-panel-header">
-    <h1>Manage Taxonomies</h1>
+    <h1>Manage Taxonomy</h1>
```

Comment:
Probably not in scope for this PR, but could we use "Vocabulary" instead of "Taxonomy"? Since we're editing at most one at a time, that would seem logical.